Source/Api/Controllers/AuthController.cs

```diff
@@ -94,34 +97,44 @@ public class AuthController : ExceptionlessApiController {
                 return Unauthorized();
             }
 
-            User user;
-            try {
-                user = await _userRepository.GetByEmailAddressAsync(model.Email);
-            } catch (Exception ex) {
-                _logger.Error().Exception(ex).Critical().Message("Login failed for \"{0}\": {1}", model.Email, ex.Message).Tag("Login").Identity(model.Email).SetActionContext(ActionContext).Write();
-                return Unauthorized();
-            }
+	        User user;
```

Comment:
Is there a way to automatically fix the spacing? I tried ctrl-k-d awhile ago and it seemed to affect more than just the lines I had touch.